test_util/helpers.py

```diff
@@ -211,3 +211,7 @@ def remove_file():
     # Attempt to remove the file upon normal interpreter exit.
     atexit.register(remove_file)
     return temp_path
+
+
+def marathon_app_id_to_mesos_dns_name(app_id):
```

Comment:
[request] - a comment on the format is desirable.. I deduce from the code that mesos dns name fro an app-id like /a/b/c will be c-b-a. A note in the docstrings would help.